### PR TITLE
Add thank you page

### PR DESCRIPTION
**Pull request by Sangarshanan**

### CODE REVIEW

File: src/content/pages/thank-you.mdx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+- Laís Carvalho
+- Jodie Burchell
+- Joana Owusu-Appiah
+- Sena Şahin
+- Sangarshanan Veera
+
+The small and big design stuff you saw including t-shirts, badges, banners, stickers and social cards were scrambled together with love by our Design team:
+
+- Panagiotis
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:49]: Full name?
Should we add Cristián here?

File: src/content/pages/thank-you.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+# Communications & Design
+
+As you browse the clear & witty content on our website, blog, newsletter, and social media, you'll be enjoying the work of our talented wordsmiths and meme creators from the Communications team.
+
+- Laís Carvalho
+- Jodie Burchell
+- Joana Owusu-Appiah
+- Sena Şahin
+- Sangarshanan Veera
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:45]: Could we add Raquel here? A lot of copy editing was done by her. I'd guess more than 60% of the manually created pages on the web have at least a minor contribution from her.

File: src/content/pages/thank-you.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+# Financial Aid team
+
+If you are one of the 221 applicants of EuroPython 2024 finaid programme then you have surely exchanged many emails with some of our Finaid team members. From spreadsheet wizardry, to responding to countless emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:
+
+- Alessia Marcolini
+- Cheuk Ting Ho
+- Vicky Twomey-Lee
+
+
+# Programme Team
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:63]: @cmaureir is this correct?

File: src/content/pages/thank-you.mdx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+From managing recording & streaming, to liaising with our wonderful remote and on-site volunteers, the Operations team have all covered. If you ever received a reply from our helpdesk, chances are you interacted with someone from this team :)
+
+- Anežka Müller
+- Angel Ramboi
+- Aris Nivorlis
+- Artem Kislovskiy
+- Cyril Bitterich
+- Daksh P. Jain
+- Ege Akman
+- George
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:95]: Full name?

File: src/content/pages/thank-you.mdx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+- Nicolás Demarchi
+- Jakub Červinka
+- Liberta Gani
+- Martin Borus
+- Moisés Guimarães
+- Niklas Mertsch
+- Panagiotis
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:102]: Full name?

File: src/content/pages/thank-you.mdx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+- Piotr Gnus
+- Raquel Dou
+- Saurav
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:105]: Full name?

File: src/content/pages/thank-you.mdx [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+
+# Sponsors Team
+
+All the sponsorship contacts, logistics & the exhibits are managed by the small team:
+
+- Angel Ramboi
+- Artem Kislovskiy
+- Artur Czepiel
+- Konstantin
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:118]: Full name?

File: src/content/pages/thank-you.mdx [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+- Moisés Guimarães
+- Niklas Mertsch
+- Panagiotis
+- Piotr Gnus
+- Raquel Dou
+- Saurav
+- Sebastiaan Zeeff
+- Shekhar Koirala
+- Theofanis Petkos
+
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:109]: This is missing the full volunteers list! @anivorlis please add.

File: src/content/pages/thank-you.mdx [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+- Mariajose Molina
+- Mia Bajić
+- Rodrigo Girao Serrao
+- Theofanis Petkos
+- Vaibhav Srivastav (VB)
+- Valerio Maggio
+- Yuliia Barabash
+
+
+# Operations Team
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:84]: @anivorlis Please check.

File: src/content/pages/thank-you.mdx [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+- Jimena
+- Karolina
+- Mariajose Molina
+- Mia Bajić
+- Rodrigo Girao Serrao
+- Theofanis Petkos
+- Vaibhav Srivastav (VB)
+- Valerio Maggio
+- Yuliia Barabash
+
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:82]: This is missing the speaker mentorship programme.

File: src/content/pages/thank-you.mdx [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:83]: @cmaureir Do we want to add the reviewers?

File: src/content/pages/thank-you.mdx [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+All the sponsorship contacts, logistics & the exhibits are managed by the small team:
+
+- Angel Ramboi
+- Artem Kislovskiy
+- Artur Czepiel
+- Konstantin
+- Naa Ashiorkor Nortey
+
+
```
Comment (cybit) [inline src/content/pages/thank-you.mdx:121]: Social Event organisers?

Comment (Sangarshanan) [reply src/content/pages/thank-you.mdx:121]: Yes, all the special event organisers are pending cause i don't know all their names, need to ping the main organisers for it

File: src/content/pages/thank-you.mdx (hunk 6 above)
Comment (Sangarshanan) [reply src/content/pages/thank-you.mdx:105]: I will need to check this another way cause I am not able to pull it out from Discord 😢

File: src/content/pages/thank-you.mdx (hunk 7 above)
Comment (aramboi) [reply src/content/pages/thank-you.mdx:118]: Konstantin Ignatov

Comment (aramboi) [inline src/content/pages/thank-you.mdx:118]:
```suggestion
- Konstantin Ignatov
```

File: src/content/pages/thank-you.mdx [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+- Panagiotis
+- Laís Carvalho
+- Patrick Arminio
+
+
+# Financial Aid team
+
+If you are one of the 221 applicants of EuroPython 2024 finaid programme then you have surely exchanged many emails with some of our Finaid team members. From spreadsheet wizardry, to responding to countless emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:
+
+- Alessia Marcolini
```
Comment (thepetk) [inline src/content/pages/thank-you.mdx:58]: @alessiamarcolini sanity check :)

File: src/content/pages/thank-you.mdx (hunk 10 above)
Comment (thepetk) [reply src/content/pages/thank-you.mdx:82]: Speaker Mentorship Programme:

Marijka Romaniuk
Naa Ashiorkor Nortey
Valery Calderón Briz

File: src/content/pages/thank-you.mdx (hunk 4 above)
Comment (thepetk) [reply src/content/pages/thank-you.mdx:95]:
```suggestion
- George Zisopoulos
```

File: src/content/pages/thank-you.mdx (hunk 5 above)
Comment (thepetk) [reply src/content/pages/thank-you.mdx:102]:
```suggestion
- Panagiotis Kyrillos
```

File: src/content/pages/thank-you.mdx (hunk 6 above)
Comment (thepetk) [reply src/content/pages/thank-you.mdx:105]:
```suggestion
- Saurav Pandey
```

File: src/content/pages/thank-you.mdx (hunk 10 above)
Comment (thepetk) [reply src/content/pages/thank-you.mdx:82]: Not sure if we would like to add all mentors too (29 mentors in total)

File: src/content/pages/thank-you.mdx (hunk 13 above)
Comment (alessiamarcolini) [reply src/content/pages/thank-you.mdx:58]: 221 + 12 (second round of remote only)

File: src/content/pages/thank-you.mdx [hunk 14]
```diff
@@ -0,0 +1,125 @@
...
+- Cheuk Ting Ho
+- Vicky Twomey-Lee
+
```
Comment (alessiamarcolini) [inline src/content/pages/thank-you.mdx:61]:
```suggestion
- Naa Ashiorkor Nortey
- Marijka Romaniuk
```

File: src/content/pages/thank-you.mdx [hunk 15]
```diff
@@ -0,0 +1,125 @@
...
+The small and big design stuff you saw including t-shirts, badges, banners, stickers and social cards were scrambled together with love by our Design team:
+
+- Panagiotis
+- Laís Carvalho
+- Patrick Arminio
+
+
+# Financial Aid team
+
+If you are one of the 221 applicants of EuroPython 2024 finaid programme then you have surely exchanged many emails with some of our Finaid team members. From spreadsheet wizardry, to responding to countless emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:
```
Comment (alessiamarcolini) [inline src/content/pages/thank-you.mdx:56]:
```suggestion
If you are one of the 233 applicants of EuroPython 2024 finaid programme then you have surely exchanged many emails with some of our Finaid team members. From spreadsheet wizardry, to responding to countless emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:
```

File: src/content/pages/thank-you.mdx [hunk 16]
```diff
@@ -0,0 +1,125 @@
...
+
+
+# Code of Conduct Committee
+
+These humans are responsible for receiving & handling Code of Conduct reports and help the EuroPython community remain a safe and friendly place:
+
+- Naomi Ceder
+- Cheuk Ting Ho
+- Lumír Balhar
+- Jakub Vysoky
```
Comment (alessiamarcolini) [inline src/content/pages/thank-you.mdx:35]: not alphabetical ordered

File: src/content/pages/thank-you.mdx (hunk 10 above)
Comment (hypha) [reply src/content/pages/thank-you.mdx:82]: I think Fanis should be there too <3

File: src/content/pages/thank-you.mdx [hunk 17]
```diff
@@ -0,0 +1,125 @@
...
+
+The folks managing Call for Proposals (CFP), community voting and the entire programme selection & scheduling are:
+
+- Artem Kislovskiy
+- Artur Czepiel
+- Cristián Maureira-Fredes
+- Cyril Bitterich
+- Diego Russo
+- Javier Lim
+- Jimena
```
Comment (cmaureir) [inline src/content/pages/thank-you.mdx:73]:
```suggestion
```
Jimena sadly dropped out the process due to not having time to participate.

File: src/content/pages/thank-you.mdx [hunk 18]
```diff
@@ -0,0 +1,125 @@
...
+- Alessia Marcolini
+- Cheuk Ting Ho
+- Vicky Twomey-Lee
+
+
+# Programme Team
+
+The folks managing Call for Proposals (CFP), community voting and the entire programme selection & scheduling are:
+
+- Artem Kislovskiy
```
Comment (cmaureir) [inline src/content/pages/thank-you.mdx:67]:
```suggestion
```
I don't remember Artem being in the process

File: src/content/pages/thank-you.mdx [hunk 19]
```diff
@@ -0,0 +1,125 @@
...
+- Artur Czepiel
+- Cristián Maureira-Fredes
+- Cyril Bitterich
+- Diego Russo
+- Javier Lim
+- Jimena
+- Karolina
+- Mariajose Molina
```
Comment (cmaureir) [inline src/content/pages/thank-you.mdx:75]:
```suggestion
- Maria José Molina-Contreras
- Cheuk Ting Ho
- Jodie Burchell
- Niklas Mertsch
- Jan Pipek
```
Those are the names that helped us with the PyData reviews.

File: src/content/pages/thank-you.mdx (hunk 3 above)
Comment (cmaureir) [reply src/content/pages/thank-you.mdx:63]: Removed two names and added missing people :+1:

File: src/content/pages/thank-you.mdx [hunk 20]
```diff
@@ -0,0 +1,125 @@
...
+The folks managing Call for Proposals (CFP), community voting and the entire programme selection & scheduling are:
+
+- Artem Kislovskiy
+- Artur Czepiel
+- Cristián Maureira-Fredes
+- Cyril Bitterich
+- Diego Russo
+- Javier Lim
+- Jimena
+- Karolina
```
Comment (cmaureir) [inline src/content/pages/thank-you.mdx:74]:
```suggestion
- Karolina Surma
```

File: src/content/pages/thank-you.mdx (hunk 1 above)
Comment (hypha) [reply src/content/pages/thank-you.mdx:49]:
```suggestion
- Panagiotis Kyrillos
```

File: src/content/pages/thank-you.mdx (hunk 11 above)
Comment (cmaureir) [reply src/content/pages/thank-you.mdx:83]: Yeah, as a category we can add them.
```suggestion
## Reviewers
* Alena Osipova
* Artur Smęt
* Daniela
* Dilyan Grigorov
* Dmitry Osipov
* Doreen Peace Nangira Wanyama
* Emma Saroyan
* Emmanuel Akanji
* Foram
* Gajendraeshpande
* George Zisopoulos
* Goetz
* HARSHIT GUPTA
* Hitesh Khandelwal
* Isaac Chung
* Jahanvi Gupta
* Jaz Allibhai
* Jiawei Wang
* Julian MAURIN
* Kalyan Prasad
* Kevin Romero
* Konstantin Ignatov
* Koti Vellanki
* Krystian
* Kshiteej
* Lajoie BATADIWA
* Leila Ben Othman
* Manav Mehra
* Mayank Khanduja
* Naa Ashiorkor Nortey
* Nancy Henrietta Addo
* Neeraj Pandey
* Oliver Thieke
* Osagie Eboigbe
* Padraic Calpin
* Patrick Arminio
* Pavel Bityukov
* Priya Basker
* Ria Bhatia
* Rodrigo Girão Serrão
* Roland Sztaho
* Sameer Shukla
* Sarah Abderemane
* Saurav Jain
* Saurav Pandey
* Sebastian M. Ernst
* Siddharth Gupta
* Simon Peteragbui
* Tim
* Tim Shchetilin
* Vamsi Krishna Kolli
* Vinícius Gubiani Ferreira
* Wachira Maina
* William
* kt zhao
* masamori0083
* shekhar
```

File: src/content/pages/thank-you.mdx (hunk 1 above)
Comment (hypha) [reply src/content/pages/thank-you.mdx:49]: Not sure if it is appropriate here, but I think the hired designers' name should be somewhere as she should be attributed for her creative work.

File: src/content/pages/thank-you.mdx (hunk 10 above)
Comment (thepetk) [reply src/content/pages/thank-you.mdx:82]: Mentors:

* Alena Osipova
* Artur Smęt
* Ashmi Banerjee
* Bernice Waweru
* Bhagvan Kommadi
* Dilyan Grigorov
* Gajendra Deshpande
* Honza Kral
* Jean-Baptiste Braun
* Jiawei Wang
* Kalyan Prasad
* Kenni Konate
* Koti Vellanki
* Lina Marieth Hoyos
* Mario García
* Max Kahan
* Maxim Danilov
* Nabanita Roy
* Neeraj Pandey
* Neil Vaytet
* Priyanshu Krishnan
* Reuven Lerner
* Ria Bhatia
* Sameer Shukla
* Sebastian Witowski
* Siddharth Gupta
* Valery Calderón Briz
* Vinícius Gubiani Ferreira
* Yuliia Barabash

File: src/content/pages/thank-you.mdx [hunk 21]
```diff
@@ -0,0 +1,125 @@
...
+- George
+- Nicolás Demarchi
+- Jakub Červinka
+- Liberta Gani
+- Martin Borus
+- Moisés Guimarães
+- Niklas Mertsch
+- Panagiotis
+- Piotr Gnus
+- Raquel Dou
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:164]:
```suggestion
```
I think you can remove me as I am not part of the awesome Ops team :)

**Review by hypha — COMMENTED** [review]
Small questions / comments that I think haven't been mentioned by other reviewers or in my previous review as replies:
1. Do you intend to add onsite volunteers here? 
2. As far as I know Panagiotis did some video work, including the awesome we are all volunteers one. I think this should either be mentioned as part of the design work or list separately. 
3. There is a lot of stuff done on the website repo by Patrick and Ege. I think this should be separately listed. 
4. Since Discord was shaped to be a fairly full-fledged team of people working on the bot and admin, I think they deserve to be specially mentioned. 
5. Isn't there an accessibility team? I think they should be listed if there is one. 
6. Did we have volunteers taking photos? They probably should be attributed.
7. I think this should be added on the homepage, as one of the CTAs ideally with a big picture of all volunteers on stage. Last year's PR as ref in case it helps: https://github.com/EuroPython/website/pull/396/

File: src/content/pages/thank-you.mdx (hunk 1 above)
Comment (hypha) [reply src/content/pages/thank-you.mdx]: Separately, the logo retained the 2023 design, so the original designers should be attributed somewhere: https://ep2023.europython.eu/thankyou/#conference-designers

Comment (Sangarshanan) [reply src/content/pages/thank-you.mdx]: Added

File: src/content/pages/thank-you.mdx (hunk 16 above)
Comment (hypha) [reply src/content/pages/thank-you.mdx:35]: Vicky Twomey-Lee  is part of the CoC team even though she could not physically be there. So I think she should be listed.

**Comment by Sangarshanan** [comment]
Thanks for the comments and wonderful review! this is in no means complete but a draft PR i raised to start the process and get feedback! I intend to add all the Onsite Volunteers, The wonderful Gonzo team, Volunteers who helped with Photographs and All the special/ social event organisers 

Answering @hypha 
1. Yes I DO! i am just trying to collate an exhaustive list
2, 3, 4, 5, 6:  YES! will mention all the teams and individual names/ shoutouts 😄
7. OMG yes i am gonna add the big picture for sure! it wanna create a poll to select the best one 🙈 

Hope this helps! please do keep commenting out suggestions and names to shoutout, i have a terribly bad memory and am so sorry if i missed mentioning any of the wonderful volunteers 

i miss you all ❤️

**Comment by hypha** [comment]
@Sangarshanan to answer your request on discord, here is the list of Gonzo team: 
- Alessandro Arese Visconti
- Ambra Lorenzetti
- Consuelo Cipriani
- Laura Pantaleo
- Matteo Piccinini
- Matteo Vanni
- Raquel Dou
- Simone Giorgi
- Vincenzo Cicanese

File: src/content/pages/thank-you.mdx (hunk 13 above)
Comment (alessiamarcolini) [reply src/content/pages/thank-you.mdx:58]: only for a brief period

File: src/content/pages/thank-you.mdx [hunk 22]
```diff
@@ -0,0 +1,293 @@
...
+- Artur Czepiel
+- Konstantin Ignatov
+- Naa Ashiorkor Nortey
+
+
+# Accessibility Team
+
+This wonderful team worked on making EuroPython 2024 accessible for everyone by setting up Quiet Rooms, Low stimulation room, Childcare and having a Neurodiversity box amongst so many other things.
+
+- Cyril Bitterich
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:246]: I think  Alan Dooley  should be listed here. If nothing else, the issues he pointed out and patiently explained has shifted our org to think and approach web accessibility for the better.

File: src/content/pages/thank-you.mdx [hunk 23]
```diff
@@ -0,0 +1,293 @@
...
+- Artem Kislovskiy
+- Artur Czepiel
+- Konstantin Ignatov
+- Naa Ashiorkor Nortey
+
+
+# Accessibility Team
+
+This wonderful team worked on making EuroPython 2024 accessible for everyone by setting up Quiet Rooms, Low stimulation room, Childcare and having a Neurodiversity box amongst so many other things.
+
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:245]:
```suggestion

- Alan Dooley 
```

File: src/content/pages/thank-you.mdx (hunk 9 above)
Comment (anivorlis) [reply src/content/pages/thank-you.mdx:205]: Done!

File: src/content/pages/thank-you.mdx (hunk 8 above)
Comment (anivorlis) [reply src/content/pages/thank-you.mdx:226]: Done.

Comment (hypha) [reply src/content/pages/thank-you.mdx:226]: It's not exactly in alphabetical order?

**Comment by anivorlis** [comment]
Hey, thanks for opening the PR ❤️ 

I made some commits for:
1. Full list of Ops team members.  I added the main sub-teams because they did significant work and should be mentioned separately. 
2. Added a list of onsite & remote volunteers
3. Added the structure for the main events & workshop, we should probably add the people behind them.
4. Some minor fixes and suggestions

Please review and suggest changes as necessary 🤗

**Review by laisbsc — COMMENTED** [review]
I stopped before the social event. Will check again later and add some more names.

File: src/content/pages/thank-you.mdx [hunk 24]
```diff
@@ -0,0 +1,457 @@
...
+# Communications & Design
+
+As you browse the clear & witty content on our website, blog, newsletter, and social media, you'll be enjoying the work of our talented wordsmiths and meme creators from the Communications team.
+
+- Joana Owusu-Appiah
+- Jodie Burchell
+- Laís Carvalho
+- Raquel Dou
+- Sangarshanan Veera
```
[the remark this answers is not in the document]
Comment (hypha) [reply src/content/pages/thank-you.mdx:45]: I think it is `Daksh P. Jain`  <3

File: src/content/pages/thank-you.mdx [hunk 25]
```diff
@@ -0,0 +1,457 @@
...
+
+# Code of Conduct Committee
+
+These humans are responsible for receiving & handling Code of Conduct reports and help the EuroPython community remain a safe and friendly place:
+
+- Cheuk Ting Ho
+- Jakub Vysoky
+- Lumír Balhar
+- Naomi Ceder
+
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:36]:
```suggestion
Vicky Twomey-Lee

```
Vicky is part of the CoC team even though she could not physically be there. So I think she should be listed.

File: src/content/pages/thank-you.mdx [hunk 26]
```diff
@@ -0,0 +1,457 @@
...
+- Laís Carvalho
+- Raquel Dou
+- Sangarshanan Veera
+- Sena Şahin
+
+The small and big design stuff you saw including t-shirts, badges, banners, stickers and social cards were scrambled together with love by our Design team:
+
+- Panagiotis Kyrillos
+- Laís Carvalho
+- Patrick Arminio
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:52]:
```suggestion
- Adriana Sanché
- Cristián Maureira-Fredes 
- Laís Carvalho
- Panagiotis Kyrillos
- Patrick Arminio
```
suggested adding Adriana and Cristián and sorting by alphabetical order.

File: src/content/pages/thank-you.mdx [hunk 27]
```diff
@@ -0,0 +1,457 @@
...
+
+
+# Speaker Mentorship Programme
+
+This team helped and supported potential speakers who want help preparing for their talk at EuroPython 2024.
+
+- Marijka Romaniuk
+- Naa Ashiorkor Nortey
+- Valery Calderón Briz
+- Theofanis Petkos
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:168]:
```suggestion
- Marijka Romaniuk
- Naa Ashiorkor Nortey
- Theofanis Petkos
- Valery Calderón Briz
```
sort by alphabetical order

File: src/content/pages/thank-you.mdx [hunk 28]
```diff
@@ -0,0 +1,457 @@
...
+- Laís Carvalho
+- Liberta Gani
+- Martin Borus
+- Niklas Mertsch
+- Nicolás Demarchi
+- Panagiotis Kyrillos
+- Piotr Gnus
+- Saurav Pandey
+- Shekhar Koirala
+- Theofanis Petkos
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:226]:
```suggestion
- Anežka Müller
- Angel Ramboi
- Aris Nivorlis
- Artur Czepiel
- Cyril Bitterich
- Ege Akman
- George Zisopoulos
- Jakub Červinka
- Laís Carvalho
- Liberta Gani
- Martin Borus
- Niklas Mertsch
- Nicolás Demarchi
- Panagiotis Kyrillos
- Piotr Gnus
- Saurav Pandey
- Shekhar Koirala
- Theofanis Petkos
```
sort by alphabetical order

File: src/content/pages/thank-you.mdx [hunk 29]
```diff
@@ -0,0 +1,457 @@
...
+A big shoutout to Ege Akman and Niklas Mertsch for leading the efforts in setting up and managing our Discord server and the discord bot!
+Their work on maintaining and updating the discord bot can be found on the [EuroPython discord bot repo](https://github.com/EuroPython/discord).
+
+The discord sub-team also included the following members:
+
+- Artur Czepiel
+- Aris Nivorlis
+- Cyril Bitterich
+- Ege Akman
+- Niklas Mertsch
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:240]:
```suggestion
- Aris Nivorlis
- Artur Czepiel
- Cyril Bitterich
- Ege Akman
- Niklas Mertsch
```
sort by alphabetical order

File: src/content/pages/thank-you.mdx [hunk 30]
```diff
@@ -0,0 +1,457 @@
...
+- Boris Kolda
+- Çağıl Uluşahin
+- Cyril Bitterich
+- Dmitry Osipov
+- Dom Weldon
+- Ege Akman
+- Elise Hannah Kasai
+- Ewa Barczykowska
+- George Zisopoulos
+- George Margaritis
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:326]:
```suggestion
- George Margaritis
- George Zisopoulos
```
sort by alphabetical order

File: src/content/pages/thank-you.mdx [hunk 31]
```diff
@@ -0,0 +1,457 @@
...
+- Haris Kupinić
+- Hugo van Kemenade
+- Iryna Kondrashchenko
+- Iva Kleban
+- Jakub Červinka
+- Jakub Balas
```
Comment (hypha) [inline src/content/pages/thank-you.mdx:332]:
```suggestion
- Jakub Balas
- Jakub Červinka
```
sort by alphabetical order

**Review by hypha — COMMENTED** [review]
minor suggestions and reordering by alphabetical order

File: src/content/pages/thank-you.mdx [hunk 32]
```diff
@@ -0,0 +1,457 @@
...
+- Aneta Popelová
+- Aris Nivorlis
+- Artur Czepiel
+- Artur Smęt
+- Barbora Hulova
+- Barbora Profantová
+- Beyza Zayim
+- Björn Brandt
+- Boris Kolda
+- Çağıl Uluşahin
```
Comment (egeakman) [inline src/content/pages/thank-you.mdx:318]:
```suggestion
- Çağıl Uluşahin Sönmez
```

File: src/content/pages/thank-you.mdx (hunk 24 above)
[the remark this answers is not in the document]
Comment (thepetk) [reply src/content/pages/thank-you.mdx:45]: Here we have Cyril Bitterich too I think. He had nice input for the team and reviewed a lot of different stuff :)

File: src/content/pages/thank-you.mdx (hunk 26 above)
Comment (thepetk) [reply src/content/pages/thank-you.mdx:52]: I would add Cyril here too :)

File: src/content/pages/thank-you.mdx [hunk 33]
```diff
@@ -0,0 +1,457 @@
...
+- Liberta Gani
+- Martin Borus
+- Saurav Pandey
+- Shekhar Koirala
+
+Special thanks to the following people for leading the onboarding and the coordination of onsite volunteers:
+
+- Anežka Müller
+- Aris Nivorlis
+- Liberta Gani
```
Comment (thepetk) [inline src/content/pages/thank-you.mdx:269]:
```suggestion
- Cyril Bitterich
- Liberta Gani
```

File: src/content/pages/thank-you.mdx [hunk 34]
```diff
@@ -0,0 +1,457 @@
...
+
+A big shoutout to Liberta and Shekhar for their exceptional work in effectively coordinating with remote speakers and in organizing remote volunteers:
+
+- Ankur Gajurel
+- Daksh P. Jain
+- Evans Cheruiyot Langat
+- Prabhash Kumar Jha
+- Pujan Thapa
+- Santosh Nepal
+- Shiva Gaire
```
[the remark this answers is not in the document]
Comment (thepetk) [reply src/content/pages/thank-you.mdx:389]: No, she was helping on site :)

File: src/content/pages/thank-you.mdx (hunk 26 above)
Comment (egeakman) [reply src/content/pages/thank-you.mdx:53]: Edited Raquel's suggestion to add the ``z`` to ``Sanché``, and added Cyril

[the remark this answers is not in the document]
Comment (egeakman) [reply src/content/pages/thank-you.mdx:56]: Done!

File: src/content/pages/thank-you.mdx (hunk 12 above)
Comment (hypha) [reply src/content/pages/thank-you.mdx:291]: Should we try to add social event organisers we know in or you've got a list somewhere? :blush:

Comment (Sangarshanan) [reply src/content/pages/thank-you.mdx:291]: I have added the names for the social event! i think id need help with the other events

**Comment by clytaemnestra** [comment]
Hi, thank you for putting the list together! 🙂 Everything looks okay to me, except of the Django Girls section. The full and correct list of coaches can be found on the DG Prague website under the Coaches section: https://djangogirls.org/en/prague/.

**Pull request merged**